platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/collections/LongListDisk.java

```diff
@@ -130,15 +130,25 @@ public void writeToFile(final Path newFile) throws IOException {
         if (!file.equals(newFile)) {
             try (final FileChannel fc =
                     FileChannel.open(newFile, StandardOpenOption.CREATE, StandardOpenOption.WRITE)) {
-                fileChannel.position(0);
-                MerkleDbFileUtils.completelyTransferFrom(fc, fileChannel, 0, fileChannel.size());
```

Comment:
This method should be significantly faster than writing a new file (header + data) from scratch. Could you comment on this change, please?

Reply:
Nevermind. I see writeLongsData() now uses completelyTransferFrom()